examples/visa/find-resources.py

```diff
@@ -0,0 +1,67 @@
+r"""Perform continuous measure record.
+
+The gRPC API is built from the C API. NI-VISA documentation is installed with the driver at:
+  C:\Program Files (x86)\IVI Foundation\VISA\WinNT\NIvisa\ni-visa.chm
+
+Getting Started:
+
+To run this example, install "NI-VISA" on the server machine:
+  https://www.ni.com/en/support/downloads/drivers/download.ni-visa.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Running from command line:
+
+Server machine's IP address, port number, and resource name can be passed as separate command line
+arguments.
+  > python find-resources.py <server_address> <port_number>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763".
+"""
+
+import sys
+
+import grpc
+import visa_pb2 as visa_types
+import visa_pb2_grpc as grpc_visa
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+
+
+# Create the communication channel for the remote host and a connection to the NI-VISA service.
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}")
+client = grpc_visa.VisaStub(channel)
+
+try:
+    # Query for resources.
+    find_resources_resp = client.FindRsrc(visa_types.FindRsrcRequest(expression="?*"))
+
+    # Get information about each resource.
+    for resource in find_resources_resp.instrument_descriptor:
+        parse_resp = client.ParseRsrcEx(visa_types.ParseRsrcExRequest(resource_name=resource))
```

Comment:
I forgot to fix this, it should be:
`parse_resp = client.ParseRsrc(visa_types.ParseRsrcRequest(resource_name=resource))`

Reply:
Commit 2